odoo/addons/base/models/ir_translation.py

```diff
@@ -161,7 +161,8 @@ def finish(self):
         cr.execute(""" INSERT INTO %s(name, lang, res_id, src, type, value, module, state, comments)
                        SELECT name, lang, res_id, src, type, value, module, state, comments
                        FROM %s AS ti
-                       WHERE NOT EXISTS(SELECT 1 FROM ONLY %s AS irt WHERE %s);
+                       WHERE NOT EXISTS(SELECT 1 FROM ONLY %s AS irt WHERE %s)
+                       ON CONFLICT DO NOTHING;
```

Comment:
That sucks.
We should fix the PO files, shouldn't we?

Reply:
The scenario in step 2 explained at odoo-dev/odoo@f9cbccd1acb9ede8f4863360f8aa797c6e788143 should probably be fixed, that's true. But I am afraid it will occur again on some db. Shouldn't the import be tolerant?